metaflow/cli.py

```diff
@@ -719,6 +719,13 @@ def wrapper(*args, **kwargs):
     hidden=True,
     help="If specified, allows this call to be called in parallel",
 )
+@click.option(
+    "--resume-identifier",
+    default=None,
+    show_default=True,
+    hidden=True,
+    help="If specified, it identifies the task that started this resume call",
```

Comment:
nit: should be something about pathspec -- I suspect we need to actually list step_name/task_id here right?

Reply:
done